GOVERNANCE.md

```diff
@@ -0,0 +1,44 @@
+NOTE: The following is a snapshot of a perpetually changing state of affairs in
+the WGPU community. It is not a binding contract between users and
+decision-makers of the WGPU project.
```

Comment:
This should be at the end - this is legaleze no user cares about.

Reply:
I think that having it at the front is the only way it'll ever be seen, but 🤷🏻‍♂️ I don't have particularly strong feelings about the position.

Reply:
This strikes me as the kind of thing that exists so we can point to when someone tries to be an ass and use this document against us.

Reply:
I think your analysis is on point. 🙂

Reply:
Moved to the bottom with 465c75ddd.